WebDevStudios/create-block
# Rename richtext demo block name to use the provided blockname

Closes #12 

### DESCRIPTION ###
This PR will rename the demo block shipped with the scaffolded plugin block to use the namespace and blockname provided as args when invoking `npm init webdevstudios/block <Namespace>/<BlockName>`

### SCREENSHOTS ###
Run the command.
![1](https://user-images.githubusercontent.com/5747475/78572521-29173d00-785a-11ea-9a8f-0d31524cd66f.png)

Demo block in editor
![2](https://user-images.githubusercontent.com/5747475/78572855-96c36900-785a-11ea-8837-81ba463fa07b.png)

![3](https://user-images.githubusercontent.com/5747475/78573112-e86bf380-785a-11ea-9f35-194b1626f935.png)

![4](https://user-images.githubusercontent.com/5747475/78573168-fcaff080-785a-11ea-879a-bf78cd94f5ee.png)

### STEPS TO VERIFY ###
1. Clone the repo.
2. Navigate inside the cloned directory.
3. Checkout the PR branch `git checkout issue/12-Refactor-rich-text-demo-block-to-use-namespace-blockname`
4. Run `npm install` 
5. Run `npm run create-block WebDevStudios/TodoBlock`
6. Move the generated `todo-block` to your WordPress' `wp-content/plugins`.
7. Activate the plugin in the Dashboard > Plugins.
8. Confirm that the demo block is named using the Namespace and blockname provided.
